pootle/apps/import_export/templates/browser/index.html

```diff
@@ -31,15 +32,19 @@
   <form action="" method="post" enctype="multipart/form-data"
     id="js-upload-form" class="upload-form">
     {% csrf_token %}
-    {% for field in upload_form %}
-      {{ field }}
-    {% endfor %}
-    <input type="submit" />
+    <div class="hide">{{ upload_form.file }}</div>
+    <span {% if not has_admin_access %}class="hide"{% endif %}>
+    {% blocktrans with user=upload_form.user_id %}
+      as {{ user }}
```

Comment:
@dwaynebailey I'm not sure I know how to do it here. "Upload translations" is a separate link.
My previous comment is related (https://github.com/translate/pootle/pull/4867#issuecomment-231297270). 

This is how it looks now:
![image](https://cloud.githubusercontent.com/assets/682845/16795165/91c91fa2-48e4-11e6-81c7-3f61cd004fe8.png)